README.md

```diff
@@ -149,3 +149,20 @@ In order to create an instance of gatekeeper in the specified namespace you can
 ```shell
 kubectl create -f config/samples/operator_v1alpha1_gatekeeper.yaml
 ```
+
+## Editing default ExemptNamespaces 
+Go to controllers/config_helper.go. Change exemptNamespaces from line 23
```

Comment:
(Optional) Use references rather than hardcoded line number; Add link to file
```suggestion
Go to [controllers/config_helper.go](controllers/config_helper.go). Update `spec.match[].exemptNamespaces` in `configDefaultResource`.
```